### ISSUE TITLE
data_sources python2.5 -- AttributeError: 'tuple' object has no attribute 'index'

### ISSUE THREAD

**Issue by GoogleCodeExporter** [jeevandongre/namebench#127]
```
What steps will reproduce the problem?
1. Run namebench.py with Python2.5 
2.
3.


What is the expected output? What do you see instead?



Python failure - use of tuple 
What version of the product are you using? On what operating system?

using namebench-1.3-RC4 on Windows XP - Python2.5

Please provide any additional information below.


  File "C:\python25\Lib\site-packages\libnamebench\data_sources.py", line 
390, in _GetSourceSearchPaths
    replacement = keywords[keywords.index(keyword)-1]
AttributeError: 'tuple' object has no attribute 'index'


```

Original issue reported on code.google.com by `marc...@gmail.com` on 30 May 2010 at 11:10

**Comment by GoogleCodeExporter** [comment]
```
This issue was closed by revision r956.
```

Original comment by `thomas.r...@gmail.com` on 30 May 2010 at 11:16
- Changed state: **Fixed**

**Comment by GoogleCodeExporter** [comment]
```
In the mean-time, the .exe downloads should work (built with python 2.7)

```

Original comment by `helixblue` on 30 May 2010 at 11:17
- Changed state: **Started**

**Comment by GoogleCodeExporter** [comment]
```
I would replace 

if sys.platform[:3] == 'win':
   keywords = ('Local', 'Roaming')

with

if sys.platform[:3] == 'win':
   keywords = ['Local', 'Roaming'] 

```

Original comment by `marc...@gmail.com` on 30 May 2010 at 11:22

**Comment by GoogleCodeExporter** [comment]
```
Fix released in 1.3-RC5.

```

Original comment by `helixblue` on 31 May 2010 at 1:29
- Changed state: **Fixed**
